cpp/src/copying/contiguous_split.cu

```diff
@@ -124,6 +127,8 @@ struct dst_buf_info {
  * @param stride Size of the kernel block
  * @param value_shift Shift incoming 4-byte offset values down by this amount
  * @param bit_shift Shift incoming data right by this many bits
+ * @param num_rows Number of rows being copied
+ * @param valid_count Optional pointer to a value to store count of unset bits.
```

Comment:
It's called "valid" count but docs say it counts unset bits, which implies it's a null count.